sample/testController.js

```diff
@@ -1,5 +1,6 @@
 //Controller for sample data
```

Comment:
Just for consistency with other files, rename `err` to `error` in catch.
Also, for `details: err`  -> must be `details: error.message`

Reply:
see [ff9f289](https://github.com/erik-lance/TitafindsXG5/pull/11/commits/ff9f28905aaea1d93ce0ee0f4de2ade76e361ff1)